### PR TITLE
Updated Extra Credit section

### PR DESCRIPTION
**Pull request by tomalley4**
- took out language around points
- added text in Brute Force to clarify the correct solution
- added period at end of last sentence, and there is still a box there that looks like an unrecognized character

### CODE REVIEW

File: python/caesar.md [hunk 1]
```diff
@@ -207,7 +205,7 @@ This is my favorite one: enhance your program's "brute force" mode so that it ca
...
 
-You can do this however you want. Be creative! My solution involved using [this text file](https://svnweb.freebsd.org/csrg/share/dict/words?revision=61569&view=co), which is a list of all of the English words in the 1934 edition of Webster's Second International Dictionary. If you'd like to figure out how to open a text file in Python and get all the lines out of it, Google around until you find a solution - you can always ask me for help if you get stuck, but I think you'd be surprised how far you can get by just Googling stuff 🙂
+You can do this however you want. Be creative! My solution involved using [this text file](https://svnweb.freebsd.org/csrg/share/dict/words?revision=61569&view=co), which is a list of all of the English words in the 1934 edition of Webster's Second International Dictionary. If you'd like to figure out how to open a text file in Python and get all the lines out of it, Google around until you find a solution - you can always ask me for help if you get stuck, but I think you'd be surprised how far you can get by just Googling stuff. 🙂
```
Comment (jrheard) [inline python/caesar.md:208]: re: unrecognized character - ha, works on my machine! https://www.evernote.com/shard/s11/sh/ba89b6f8-6b58-438a-be55-ef7059f721df/ea381d04cc519079

i assume that when you were looking at this just now, you were on one of the computers in your classroom? if so, i'll just go ahead and change this to a regular :) smiley face - according to http://caniemoji.com/microsoft-windows/ , apparently chrome can't display emojis on windows < 10, which is surprising but oh well.

**Review by jrheard — APPROVED** [review]

File: python/caesar.md (hunk 1 above)
Comment (tomalley4) [reply python/caesar.md:208]: Yep, I'm at school. It's a mac vs pc issue I bet...

Comment (tomalley4) [reply python/caesar.md:208]: I get a lot of J's in my emails, which I think is the same issue but a different "unknown character" translation.

Comment (jrheard) [reply python/caesar.md:208]: yeah, that mystified me for years, i kept getting inexplicable Js in emails from business-type-y folks. i looked it up at one point and it turns out it's a bug in microsoft outlook. http://www.businessinsider.com/microsoft-outlook-fixes-j-smiley-emoticon-emoji-bug-2017-5 has details, sounds like they finally got around to publishing a fix for it seven years later, so hopefully that issue will go away over time as people update their copies of outlook!

**Pull request merged**